### ISSUE TITLE
Mkdir race conditions inspection is not reported anymore

### ISSUE THREAD

**Issue by Koc** [kalessil/phpinspectionsea#691]
```
PhpStorm 2017.3 EAP
Build #PS-173.3727.84, built on November 22, 2017
PhpStorm EAP User
Expiration date: December 22, 2017
JRE: 1.8.0_152-release-1024-b6 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
```

![default](https://user-images.githubusercontent.com/191082/33993294-9713afb6-e0de-11e7-994c-e7e0e7b7f5cb.png)

![default](https://user-images.githubusercontent.com/191082/33993335-b778b42c-e0de-11e7-9d86-4911cf217975.png)

**Comment by kalessil** [comment]
Fixed, for some reason, I was checking only calls with one argument.
